CMakeLists.txt

```diff
@@ -13,6 +13,40 @@ set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
+
+set(DEFAULT_COMPILE_OPTIONS)
+# MSVC compiler options
+if ("${CMAKE_CXX_COMPILER_ID}" MATCHES "MSVC")
+    set(DEFAULT_COMPILE_OPTIONS ${DEFAULT_COMPILE_OPTIONS}
+    PRIVATE
+        /MP           # -> build with multiple processes
+        /W4           # -> warning level 4
+        # /WX         # -> treat warnings as errors
+        /wd4251       # -> disable warning: 'identifier': class 'type' needs to have dll-interface to be used by clients of class 'type2'
+        /wd4592       # -> disable warning: 'identifier': symbol will be dynamically initialized (implementation limitation)
+        # /wd4201     # -> disable warning: nonstandard extension used: nameless struct/union (caused by GLM)
+        /wd4127       # -> disable warning: conditional expression is constant (caused by Qt)
+
+        # /Zm114      # -> Memory size for precompiled headers (insufficient for msvc 2013)
+        /Zm200        # -> Memory size for precompiled headers
+        
+        #$<$<CONFIG:Debug>:
+        #/RTCc         # -> value is assigned to a smaller data type and results in a data loss
+        #>
+
+        $<$<CONFIG:Release>: 
+        /Gw           # -> whole program global optimization
+        /GS-          # -> buffer security check: no 
+        /GL           # -> whole program optimization: enable link-time code generation (disables Zi)
+        /GF           # -> enable string pooling
+        >
+        
+        # No manual c++11 enable for MSVC as all supported MSVC versions for cmake-init have C++11 implicitly enabled (MSVC >=2013)
+
+        PUBLIC
+    )
+endif ()
```

Comment:
This is out of scope of this PR and a solution proposed in #804

Reply:
I don't see why it is out of the scope of the PR, as this code is added in the proposal.

Reply:
The PR #804 proposes what you suggest (and I agree on this suggestion) here in a more complete solution. For now, and as this is not required to build Radium libs and example apps, including test, on windows (tested -- even if it helps accelerating build and minimizing warnings in CI), we can remove this from this PR and, later, work on the global proposal of PR #804. That's why I think it is out of scope of this PR as it will propose only an incomplete solution.

Reply:
Ok so I suggest we remove these changes as they are not required here.